pizza-time/src/components/card/card.js

```diff
@@ -62,7 +64,7 @@ class Card extends React.Component {
     console.log("Event neing posted ", newItem);
 
     axios
-      .put(`http://localhost:5500/api/events/status/${id}`, newItem)
+      .put("http://" + this.state.host + "/api/events/status/" + id, newItem)
```

Comment:
Here you could use the template literal syntax `https://${this.state.host}/api/events/status/${id}`